qemu/tests/spapr_vty_multi_backends.py

```diff
@@ -27,7 +27,7 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.wait_for_login()
     for serial_id in params.objects("serials"):
-        if serial_id != "vs1":
+        if serial_id != "vs1" and serial_id != "vs9":
```

Comment:
Any checkpoint of `vs9`?

Reply:
According to the test case, where the 9th or larger number spapr-vty devices could not be used as serial since the maximum available console/serial devices is 8 inside the guest, i.e. from /dev/hvc0 to /dev/hvc7, it will be not added here.